OAT/Operations/RegexOperation.cs

```diff
@@ -54,9 +54,20 @@ internal OperationResult RegexOperationDelegate(Clause clause, object? state1, o
             (var stateTwoList, _) = Analyzer?.ObjectToValues(state2) ?? (new List<string>(), new List<KeyValuePair<string, string>>());
             if (clause.Data is List<string> RegexList && RegexList.Any())
             {
+                var options = RegexOptions.Compiled;
+                
+                if (clause.Arguments.Contains("i"))
+                {
+                    options |= RegexOptions.IgnoreCase;
+                }
+                if (clause.Arguments.Contains("m"))
+                {
+                    options |= RegexOptions.Multiline;
+                }
+
                 var built = string.Join("|", RegexList);
 
-                var regex = StringToRegex(built);
+                var regex = StringToRegex(built,options);
```

Comment:
Nit, formatting.

Reply:
Thanks for the catches.  Ran the formatter to resolve all.